src/main/java/uk/gov/hmcts/reform/sendletter/cache/SentLettersCache.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.sendletter.cache;
+
+import org.redisson.api.RSetCache;
+import org.redisson.api.RedissonClient;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sendletter.model.Letter;
+import uk.gov.hmcts.reform.sendletter.services.LetterChecksumChecker;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Stores (successfully) sent letters for predefined amount of time.
+ */
+@Component
+public class SentLettersCache {
```

Comment:
I think we can rename this as `LettersCache` instead as we won't know whether letters were successfully sent by notify client.

Reply:
we do, if notify call failed, we remove the letter from cache

Reply:
ok. I was thinking from the perspective that even if the notify call succeeded from our side we actually don't know whether the letters were actually printed and sent by notifiy client.